src/Controls/src/Core/HandlerImpl/SwipeView/SwipeView.iOS.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Microsoft.Maui.Controls
+{
+	public partial class SwipeView
+	{
+		MauiSwipeView? _mauiSwipeView;
+
+		private protected override void OnHandlerChangedCore()
+		{
+			base.OnHandlerChangedCore();
+
+			if (Handler is not null)
+			{
+				if (Handler is SwipeViewHandler swipeViewHandler && swipeViewHandler.PlatformView is MauiSwipeView mauiSwipeView)
+				{
+					_mauiSwipeView = mauiSwipeView;
+					_mauiSwipeView.SwipeItemsAdded += OnSwipeItemsAdded;
+					_mauiSwipeView.SwipeItemsRemoved += OnSwipeItemsRemoved;
+				}
+			}
+			else if (_mauiSwipeView is not null)
+			{
+				_mauiSwipeView.SwipeItemsAdded -= OnSwipeItemsAdded;
+				_mauiSwipeView.SwipeItemsRemoved -= OnSwipeItemsRemoved;
+				_mauiSwipeView = null;
+			}
+		}
+
+		void OnSwipeItemsAdded(object? sender, SwipeItemsAddedEventArgs e)
+		{
+			foreach (var swipeItem in e.AddedSwipeItems)
```

Comment:
SwipeItems are a special case because are added and removed dynamically when open and close the SwipeView. We will add to the LogicalChildren collection the visible SwipeItems when opening the SwipeView and remove them on closing.

Reply:
After adding the children to the `LogicalChildren `collection, should appear in the Live Visual Tree?